### PR TITLE
Ci template

### PR DESCRIPTION
**Pull request by smousa**

### CODE REVIEW

**Comment by smousa** [comment]
Should I add a README.md to the ci directory or should I modify the root README.md to provide steps on how to add your service pipeline in jenkins?

File: template/Makefile [hunk 1]
```diff
@@ -24,14 +24,19 @@ GOBINDATA            := $(shell command -v go-bindata 2> /dev/null)
 ifndef GOBINDATA
 GOBINDATA            := $(DOCKER_CMD) go-bindata
 endif
+IMAGE_NAME			 := zenoss/{{Name}}
```
Comment (iancmcc) [inline template/Makefile]: This should be `zenoss/zing-{{Name}}`, or we need to change other things

File: template/ci/service.groovy [hunk 2]
```diff
@@ -0,0 +1,2 @@
+DOCKER_IMAGE=zenoss/{{Name}}
```
Comment (iancmcc) [inline template/ci/service.groovy]: Ditto with the zing thing

File: template/ci/.gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.project_name
+.image_tag
+vendor.yaml
```
Comment (iancmcc) [inline template/ci/.gitignore]: This should be `version.yaml`

File: template/ci.mk [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+.PHONY: push
+push: $(CI_IMAGE_TAG)
+ifndef REMOTE_IMAGE
+	@echo "REMOTE_IMAGE not set" 1>&2; exit 2
+else
+	@docker tag $(IMAGE_NAME) $(REMOTE_IMAGE)
+	@docker push $(REMOTE_IMAGE)
+endif
+
+version:
```
Comment (iancmcc) [inline template/ci.mk]: This target should be `ci/version.yaml`

**Pull request merged**